### PR TITLE
Micro reddit modeling project

### PR DESCRIPTION
**Pull request by marcelomaidden**
## Micro Reddit modeling project
At this pull request, we create the models below and all the relationships and validations needed:
- Users
- Posts
- Comments

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @marcelomaidden and @mke2111 
### You have done well so far :clap: However, you have a few issues to fix:-
- [ ] In the validation from models, only validate a field in one line and separate more conditions with a comma. See below for more comments
### Make the changes and submit for another code review. Good luck.
- Happy coding!

File: app/models/comment.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  validates :description, presence: true
+  validates :description, length: { in: 2..500 }
```
Comment (rOluochKe) [inline app/models/comment.rb]: ### Put this in one line and separate the conditions with a comma

File: app/models/post.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+
+  validates :link, presence: true
+  validates :link, length: { maximum: 500 }
```
Comment (rOluochKe) [inline app/models/post.rb]: ### Put this in one line and separate the conditions with a comma

File: app/models/user.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class User < ApplicationRecord
+  validates :username, presence: true
+  validates :username, uniqueness: true
+
+  validates :email, presence: true
+  validates :email, confirmation: true
+  validates :email, uniqueness: true
```
Comment (rOluochKe) [inline app/models/user.rb]: ### Put this in one line and separate the conditions with a comma

**Review by ghost — APPROVED** [review]
# Status: Approved

**Congratulations team and good luck in your next project 😃.**

**Pull request merged**